### ISSUE TITLE
Any reason why check circle outline is not present in Material Icons ?

### ISSUE THREAD

**Issue by olup** [react-icons/react-icons#300]
This is a very simple question / feature request. Material design has a CheckCircle and a CheckCircleOutline icon. React icons however only exports MdCheckCircle (the filled one). Other circle icons have both versions. Any reason why not CheckCircle ?

**Comment by kamijin-fanta** [comment]
I couldn't find it. I want an SVG link on Github.
https://github.com/google/material-design-icons

**Comment by olup** [comment]
You are right, their github repo is way out of sync with their icon
specification page. However, material-ui has the totality of icons in their
repo - only in a js form. If I can extract the svg of all icons from their
project, and publish that on github, would that make it possible to include
them all in react-icon ?

Le mer. 22 avr. 2020 à 14:17, kamijin_fanta <notifications@github.com> a
écrit :

**Comment by kamijin-fanta** [comment]
Yes it is possible. But, currently planning to use only the sources provided by the official GitHub repo or npm package. This is because inconsistencies occurred in the version upgrade in the past.
